graph/src/blockchain/mod.rs

```diff
@@ -510,18 +512,36 @@ impl BlockchainKind {
 
 /// A collection of blockchains, keyed by `BlockchainKind` and network.
 #[derive(Default, Debug, Clone)]
-pub struct BlockchainMap(HashMap<(BlockchainKind, String), Arc<dyn Any + Send + Sync>>);
+pub struct BlockchainMap(pub HashMap<(BlockchainKind, ChainId), Arc<dyn Any + Send + Sync>>);
```

Comment:
Do we need to make this `pub`? Could we get by with adding some accessor method to `BlockchainMap` instead?

Reply:
added an iter accessor